indico/modules/events/timetable/templates/pdf/_program.html

```diff
@@ -0,0 +1,340 @@
+{% macro render_program(event, days, config) %}
+    {% for day in days %}
+        <div class="day pagebreak program-{{loop.index}} {{ 'pagebreak-children' if config.new_page_per_session else '' }}" id="day-{{ loop.index }}">
+            <div class="day-title">
+                <h2>{{ day|format_skeleton('EEEEdMMMM', timezone=event.timezone) }}</h2>
+            </div>
+            {% for entry in days[day] %}
+                {% set should_render = (
+                    entry.type.name == 'SESSION_BLOCK' or
+                    (entry.type.name == 'CONTRIBUTION' and config.show_contribs) or
+                    (entry.type.name == 'BREAK' and config.show_breaks)
+                ) %}
+
+                {% if should_render %}
+                    <div class="day-entry">
+                        {% if entry.type.name != 'CONTRIBUTION' %}
+                            {{ get_color_styles(entry) }}
+                        {% endif %}
+                        <div class="time-text time-from">
+                            {{ entry.start_dt|format_time(timezone=event.timezone) }}
```

Comment:
You can use `event.tzinfo` in most of those places; those date formatting utils accept both a string and a tzinfo object, but at least to me it feels nicer to pass around tzinfos instead of string timezones